src/LiveComponent/assets/dist/live.min.css

```diff
@@ -1 +1 @@
-[data-loading=""],[data-loading="delay|show"],[data-loading=show]{display:none}
\ No newline at end of file
+[data-loading=""],[data-loading=show],[data-loading=delay\|show]{display:none}
```

Comment:
@Kocal  Not sure about that... and we may want to discuss this kind of thing together 😅